pos_global_discount_in_line/models/pos_config.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    global_discount_in_line = fields.Boolean(
+        string="Add Discount in Line", default=False
```

Comment:
```suggestion
        string="Add Discount in Line"
```

it's already `False` by default

Reply:
It is null by default. At least from the point of database there is difference between null and false.